### DISCUSSION TITLE
Many-to-any navigation property

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#4520] [Feature Requests]
This is a suggestion to have navigation properties (eg. alias field) on the "any" collection in an m2a relationship, that allow finding/listing all the item's "many" parents in an interface silimar to the o2m.

**Comment by rijkvanzanten** [comment]
Is this what you meant by that one-to-any interface discussion? https://github.com/directus/directus/discussions/4525

**Reply by ghost** [reply]
No this is a separate issue. This issue is about allowing a property on the "any" collection that allows finding out all its "many" collections. Eg. 1 article can have any content block types. Right now I can use an m2a interface to find out all the article's content blocks and their types. However, looking at any one of those content block items separately, there is no interface to find out all its related articles. Hopefully that's clearer?
